chapters/08-documentation.Rmd

```diff
@@ -4,7 +4,49 @@
 - Function documentation (with roxygen2), part 2
 - Running spell checks and styling (with spelling and styler)
 
+## Learning Objectives
 
+* Describe the purpose of documenting code and packages
+* Document functions with `roxygen`
+* Create a README and vignette for package documentation
+* Ensure that documentation is consistent with up-to-date style guidelines using `styler`
+
+## Outline
+
+* Introduction: the purpose of documentation
```

Comment:
Are talking as well of documenting tests or only functions?

Reply:
Since tests are covered in chapter 6, I think it makes sense to talk about documenting tests here as well. Forgive my R ignorance, but would `roxygen` also be used for tests? Or would it be more inline comments?

Reply:
Tests would be documented with inline comments since roxygen is only for functions and building the user-facing package documentation. Generally though, the way tests are written, you don't need to add much extra documentation aside from what is required as part of testthat package setup.